<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>531</width>
      <height>198</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="User_label">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="username_line">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="Pass_label">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="password_line">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="font">
        <font>
         <family>MS PGothic</family>
         <pointsize>13</pointsize>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="or_label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Or</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="signupButton">
       <property name="font">
        <font>
         <family>MS PGothic</family>
         <pointsize>13</pointsize>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>Signup</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
